storybook/stories/API/shapes/Cross.mdx

```diff
@@ -0,0 +1,8 @@
+import { ArgTypes } from '@storybook/blocks';
+import * as CrossStories from './Cross.stories';
+
+# Cross
+
+TODO
```

Comment:
```suggestion
The Cross is a shape that itself is not a valid child in any component. 
It can be used to implement custom elements, returning it from within a

\```jsx
<Customized component={<Cross {...crossProps} />}>
\```

or other custom components (i.e. shape of a bar) 

```